site/plugins.md

```diff
@@ -390,8 +391,245 @@ func (p *Plugin) Report(w http.ResponseWriter, r *http.Request) {
 
 ```
 
-**See Also**
+### <a id="report-data-structures"></a>Report Data structures
+A report can contain many types of information.
+If you go back to the [Reporter Interface](#reporter-interface) section, you see the top-level `Plugins` attribute.
+Along with that, a report may contain multiple topologies.
+An example of a report containing few topologies is the following:
 
-  * [Building Scope](/site/building.md)
+```json
+{
+  "Host": {},
+  "Container": {},
+  "Process": {},
+  ...,
+  "Plugins": [...,]
+}
+```
+
+### Topologies
+Topologies can be of various types.
+Each topology consists of a list of nodes and other information about the topology.
```

Comment:
How about "Each topology consists of some general information and a list of nodes."?